backend/planner.py

```
@@ -118,12 +118,6 @@ def generate_plan(cluster: IssueCluster, feedback_items: list[FeedbackItem]) ->
 
     except Exception as e:
```

Ruff (0.14.10)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)